src/PIL/TiffImagePlugin.py

```diff
@@ -1416,7 +1421,16 @@ def _setup(self) -> None:
         if not isinstance(xsize, int) or not isinstance(ysize, int):
             msg = "Invalid dimensions"
             raise ValueError(msg)
-        self._size = xsize, ysize
+        self._will_be_transposed = self.tag_v2.get(ExifTags.Base.Orientation) in (
+            5,
+            6,
+            7,
+            8,
+        )
```

Comment:
Just a stylistic correction
```suggestion
        orientation = self.tag_v2.get(ExifTags.Base.Orientation)
        self._will_be_transposed = orientation in (5, 6, 7, 8)
```